### PR TITLE
[API] Set context of backend enabled

### PR DESCRIPTION
**Pull request by dayo09**
This commit sets vscode context of enabled backend.

ONE-vscode-DCO-1.0-Signed-off-by: Dayoung Lee <dayoung.lee@samsung.com>

### CODE REVIEW

File: src/Backend/API.ts [hunk 1]
```diff
@@ -29,20 +29,33 @@ interface BackendMap {
...
+}
+
 const registerBackend = (backend: Backend) => {
   const backendName = backend.name();
   assert(backendName.length > 0);
   globalBackendMap[backendName] = backend;
   const compiler = backend.compiler();
   if (compiler) {
     gToolchainEnvMap[backend.name()] = new ToolchainEnv(compiler);
+    vscode.commands.executeCommand(
```
Comment (dayo09) [inline src/Backend/API.ts:54]: FYI, this is the context that can be referred in package.json

File: src/Backend/API.ts [hunk 2]
```diff
@@ -29,20 +29,33 @@ interface BackendMap {
...
 let globalExecutorArray: Executor[] = [];
 
 const logTag = "API";
 
+// TODO Move outside API.ts
+export namespace BackendContext {
+  export const isRegistered = (backendName: string) => {
+    return globalBackendMap[backendName] !== undefined;
+  };
+}
```
Comment (dayo09) [inline src/Backend/API.ts:45]: FYI, this is the context that can be referred in any modules.

File: src/Backend/API.ts (hunk 1 above)
Comment (jyoungyun) [reply src/Backend/API.ts:54]: There is no `getContext` API in vscode. Therefore, both apis should be provided to get the backend status.

**Review by jyoungyun — APPROVED** [review]
LGTM

**Review by profornnan — APPROVED** [review]
It seems like it will be helpful for modularization. Thank you 😄

**Review by rhqjatn2398 — APPROVED** [review]
LGTM

**Comment by hohee-hee** [comment]
LGTM
We'll modulize EdgeTPU Compiler based on this context😊

**Review by hohee-hee — APPROVED** [review]
LGTM
We'll modulize EdgeTPU Compiler based on this context😊

**Pull request merged**